worlds/osu/Client.py

```diff
@@ -520,7 +487,41 @@ async def download_next_beatmapset_silent(ctx, task):
     except Exception as e:
         print(f"An error occurred: {e}")
 
+# This is the non-silent version of the function above where this one is used in the update command
+async def download_next_beatmapset(self, task):
```

Comment:
having this be its own function is kinda akward but you're already planning a refactor so its fine

Reply:
I thought it would be out of scope for this PR to fix it, next one will fix it